jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/sam/SamCommon.kt

```diff
@@ -112,7 +113,7 @@ class SamCommon {
                 ?: return "UNKNOWN"
 
             return try {
-                SamVersionCache.evaluateBlocking(sanitizedPath).result.rawVersion
+                SamVersionCache.evaluateBlocking(sanitizedPath, SamVersionCache.DEFAULT_TIMEOUT_MS).result.rawVersion
```

Comment:
Should add a comment so we don't accidentally remove this timeout

Reply:
Added a big doc comment